index.html

```diff
@@ -1,75 +1,420 @@
+								<button class="hosting-plan-2__button-2">
+									Dodaj do koszyka
+								</button>
+							</div>
+							<p class="hosting-plan-2__price-reneval">
+								8,99 zł/mies. po odnowieniu
+							</p>
+							<div class="hosting-plan-2__advantages">
+								<div class="hosting-plan-2__advantages-all">
+									<img src="./svg/yellow-bird.svg" alt="" />
+									<p><strong>1</strong> strona internetowa</p>
+								</div>
+								<div class="hosting-plan-2__advantages-all">
+									<img src="./svg/yellow-bird.svg" alt="" />
+									<p>
+										<strong>30 GB</strong> pamięci masowej
+										SSD
+									</p>
+								</div>
+								<div class="hosting-plan-2__advantages-all">
+									<img src="./svg/yellow-bird.svg" alt="" />
+									<p>
+										<strong>~10000</strong> wizyt
+										miesięcznie
+									</p>
+									<img
+										class="hosting-plan-2__help-outline"
+										src="./svg/help-outline.svg"
+										alt=""
+									/>
+								</div>
+								<div class="hosting-plan-2__advantages-all">
+									<img src="./svg/yellow-bird.svg" alt="" />
+									<p><strong>1</strong> konto e-mail</p>
+								</div>
+								<div class="hosting-plan-2__advantages-all">
+									<img src="./svg/bird.svg" alt="" />
+									<p>
+										<strong>Darmowy</strong> SSL (cena 45,25
+										zł)
+									</p>
+									<img
+										class="hosting-plan-2__help-outline"
+										src="./svg/help-outline.svg"
+										alt=""
+									/>
+								</div>
+								<div class="hosting-plan-2__advantages-all">
+									<img src="./svg/red-cross.svg" alt="" />
+									<p><strong>Darmowa</strong> domena</p>
+								</div>
+								<div class="hosting-plan-2__advantages-all">
+									<img src="./svg/yellow-bird.svg" alt="" />
+									<p>
+										Przepustowość na poziomie<strong
+											>100 GB</strong
+										>
+									</p>
+								</div>
+								<div class="hosting-plan-2__advantages-all">
+									<img src="./svg/yellow-bird.svg" alt="" />
+									<p>
+										<strong>Zarządzany</strong> system
+										WordPress
+									</p>
+								</div>
+							</div>
+							<hr class="hosting-plan-2__line" />
+							<div class="hosting-plan-2__see-all-adventages">
+								<p>Zobacz wszystkie funkcje</p>
+								<img src="./svg/navigation-header.svg" alt="" />
+							</div>
+						</div>
+					</div>
+				</container>
+				<container class="hosting-plan-3">
+					<div class="hosting-plan-3__table">
+						<div class="hosting-plan-3__column-1">
+							<strong class="hosting-plan-3__text-top-1"
+								>Hosting Single</strong
+							>
+							<p class="hosting-plan-3__text-top-2">
+								Idealne rozwiązanie dla początkujących
+							</p>
+							<div class="hosting-plan-3__price">
+								<p class="hosting-plan-3__price-text">
+									25,99 zł
+								</p>
+								<div>
+									<button class="hosting-plan-3__button-1">
+										<strong>Rabat 85%</strong>
```

Comment:
A po co ten strong? Jak chcesz pogrubienie, to dodaj wlasciwosc css do guzika. Strong ma znaczenie semantyczne (mowi googlowi ze ta czesc kodu jest wazna)